validator/src/cli.rs

```diff
@@ -343,14 +343,24 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .help(use_snapshot_archives_at_startup::cli::HELP)
                 .long_help(use_snapshot_archives_at_startup::cli::LONG_HELP),
         )
+        .arg(
+            Arg::with_name("full_snapshot_archive_path")
+                .long("full-snapshot-archive-path")
+                .value_name("DIR")
+                .takes_value(true)
+                .help(
+                    "Use DIR as full snapshot archives location \
+                     [default: --snapshots value]",
```

Comment:
If none of the args are set, `--snapshots` is `LEDGER_DIR/snapshots`. But, `--full-snapshot-archive-path` would actually be `LEDGER_DIR` without the extra `snapshots` subdirectory. So the default value listed here is technically incorrect right ?

Reply:
> I ended up settling on the idea that --snapshots is really setting a "base" snapshots directory to be used by the three real snapshot paths: full snapshot archives, incremental snapshot archives, and bank snapshots.

Agreed, I also came to a similar conclusion running through the code again

> Something like:
>>Use DIR as the base location for snapshots. A subdirectory named "snapshots" will be created. [default: --ledger value]

I like it and I think it conveys the idea of this being a "base" directory